README.md

```diff
@@ -1,16 +1,54 @@
-# [Project Name]
+# Python Adventures
 
-[project description]
+![Alt title](bot/assets/title-art.png)
 
+Python Adventures is a discord game bot where you answer pythonic questions and gain rewards.
 
-## How to prepare the project?
+# Installation
+## Discord API Token
+What is a Discord API Token?
+A Discord API Token is a unique identifier used to authenticate requests to the Discord API. It acts as a password for your bot, allowing it to interact with Discord's servers, join channels, send messages, and perform other actions as defined by the Discord API.
 
-(optional) Create a virtual environment for this project.
+## How to Obtain a Discord API Token
+1. Create a New Application:
+  - Go to the [***Discord Developer Portal***](https://discord.com/developers/applications)
+  -  Click on ***New Application***
+  -   Give your application a name and click **Create**.
 
-1. Install the necessary packages: `pip install -r requirements.txt`
+2. Create a Bot:
+  - Navigate to the ***Bot*** section in the sidebar.
+  -  Click on "Add Bot" and confirm by clicking ***Yes, do it!***
 
-2. Create a `.env` at the root of the project, if not available.
+3. Copy the Token:
+   - Under the "TOKEN" section, click "Copy" to copy your bot's token.
+   - Keep this token secure and never share it publicly. If your token is exposed, you should regenerate it immediately.
 
-3. add or set `DISCORD_BOT_KEY=[your_bot_key]` with bot's key.
+For more information, Consult The [Discord Developer Portal](https://discord.com/developers/docs/intro) Documentation.
```

Comment:
We also have a dev guide for setting up the bot in discord's dev portal. We should either link to that or integrate that text into the README. IMO integrating it into the readme would make the most sense.